libs/component-library/src/lib/card/card.tsx

```diff
@@ -69,3 +72,35 @@ export const FxCard = ({
     />
   );
 };
+
+const FxCardTitle = (props: FxTextProps) => (
+  <FxText color="content1" variant="bodyLargeRegular" {...props} />
+);
+
+const FxCardRow = (props: FxBoxProps) => (
```

Comment:
Should we remove the components I created in the box app?

Reply:
I was actually planning on updating it with this one once merged... so Happy to do that next.